app/controllers/bulk_actions_controller.rb

```diff
@@ -2,6 +2,8 @@
 
 class BulkActionsController < ApplicationController
   before_action :set_bulk_action, only: [:destroy, :file]
+  include Blacklight::Catalog
+  copy_blacklight_config_from CatalogController
```

Comment:
I don't understand why this is necessary. Can you explain?

Reply:
This is needed so I can render the selected search with `render_constraint` in the create job view...unless there is a more clever way